docs/source/gbd2019_models/causes/ihd/index.rst

```diff
@@ -4,35 +4,36 @@
 Ischemic Heart Disease
 ======================
 
-.. todo::
-
-  Add a brief introductory paragraph for this document.
-
 .. contents::
    :local:
    :depth: 1
 
-.. list-table:: Abbreviations
-  :widths: 15 15 15
-  :header-rows: 1
 
-  * - Abbreviation
-    - Definition
-    - Note
-  * - 
-    - 
-    - 
+.. list-table:: Abbreviations
+   :widths: 15 15 15
+   :header-rows: 1
 
-.. todo::
+   * - Abbreviation
+     - Definition
+     - Notes
+   * - IHD
+     - Ischemic heart disease
+     - 
+   * - MI
+     - Myocardial infarction
+     - AMI refers to Acute MI
```

Comment:
Might as well give AMI its own row in the abbreviations table

Reply:
Done! Thanks, Abie!